packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -248,11 +248,17 @@ export class FlowBuilderApi {
     if (this.flow.locales.find(flowLocale => flowLocale === locale)) {
       return locale
     }
-    const language = locale.split('-')[0]
-    if (this.flow.locales.find(flowLocale => flowLocale === language)) {
+
+    const language = locale?.split('-')[0]
```

Comment:
you could also do:
`const [language, _] = locale?.split('-')` and you will already have it